src/components/banner/banner.less

```diff
@@ -91,9 +125,15 @@
 							transition: max-height 0.5s cubic-bezier(0, 1, 0, 1);
 
 							ul {
-								margin: 5px 0 15px 0;
+								margin: 5px 0 5px 0;
 								padding: 0 0 0 36px;
 							}
+
+							@media @smartphone {
+								ul {
+									padding-left: 15px;
+								}
+							}
```

Comment:
curious why indentation so crazy in these .less files. assuming it was this way in the third-party library we cloned

Reply:
yeah spacing all over the place in these files and maintainer gitignored all the vs code stuff, hoping to get this corrected over time